### PR TITLE
docs: document how to get kubelet logs with Bottlerocket

### PR DESCRIPTION
**Pull request by tzneal**
**Description**

Update docs for getting Kubelet logs on AL2 and add docs for Bottlerocket.

**How was this change tested?**

*

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/troubleshooting.md [hunk 1]
```diff
@@ -33,17 +33,34 @@ Reasons that a node can fail to join the cluster include:
 - Security Groups
 - Networking
 
-The easiest way to start debugging is to connect to the instance
-```sh
+The easiest way to start debugging is to connect to the instance and get the Kubelet logs.  For an AL2 based node:
```
Comment (jonathan-innis) [inline website/content/en/preview/troubleshooting.md:36]: Consider pointing to the installation of the SSM manager as a DS [here](https://docs.aws.amazon.com/prescriptive-guidance/latest/patterns/install-ssm-agent-on-amazon-eks-worker-nodes-by-using-kubernetes-daemonset.html)

Comment (bwagner5) [reply website/content/en/preview/troubleshooting.md:36]: might be confusing to post that link since AL2 has the SSM Agent already installed by default

**Review by bwagner5 — APPROVED** [review]
lgtm

File: website/content/en/preview/troubleshooting.md (hunk 1 above)
Comment (jonathan-innis) [reply website/content/en/preview/troubleshooting.md:36]: Ah, didn't know that, good to know :)

**Pull request merged**